src/cli/cli_dataset.cpp

```diff
@@ -602,13 +602,41 @@ otError Dataset::Print(otOperationalDatasetTlvs &aDatasetTlvs)
         {"Security Policy", "securitypolicy"},
     };
 
-    otError              error;
-    otOperationalDataset dataset;
+    static const ComponentTitle kNonsensitiveTitles[] = {
+        {"Pending Timestamp", "pendingtimestamp"},
+        {"Active Timestamp", "activetimestamp"},
+        {"Channel", "channel"},
+        {"Wake-up Channel", "wakeupchannel"},
+        {"Channel Mask", "channelmask"},
+        {"Delay", "delay"},
+        {"Ext PAN ID", "extpanid"},
+        {"Mesh Local Prefix", "meshlocalprefix"},
+        {"Network Name", "networkname"},
+        {"PAN ID", "panid"},
+        {"Security Policy", "securitypolicy"},
```

Comment:
Will this work:
1. Add a new boolean 'mIsSensitive' with false as the default value. 
2. Skip the sensitive lines when printing the dataset.

Reply:
Done.